doc/manual/strings.rst

```diff
@@ -349,12 +349,9 @@ exception handling required:
     <BLANKLINE>
     y
 
-UTF-8 is not the only encoding that Julia supports, and adding support
-for new encodings is quite easy.  In particular, Julia also provides
-:obj:`UTF16String` and :obj:`UTF32String` types, constructed by
-:func:`utf16` and :func:`utf32` respectively, for UTF-16 and
-UTF-32 encodings. Additional discussion of other encodings and how to
-implement support for them is beyond the scope of this document for
+Julia uses UTF-8 encoding by default, and support for new encodings can
+be added by packages. Additional discussion of other encodings and how
```

Comment:
Shouldn't it say that the UTF-16 and UTF-32 encodings are supported by the LegacyStrings package, since the latter is a quasi-official package mentioned in the deprecation warnings?